src/cts/test/check_buffers_blockages.ok

```diff
@@ -12,7 +12,7 @@
 [INFO CTS-0010]  Clock net "clk" has 288 sinks.
 [INFO CTS-0008] TritonCTS found 1 clock nets.
 [INFO CTS-0097] Characterization used 1 buffer(s) types.
-[INFO CTS-0200] 5 placement blockages have been identified.
+[INFO CTS-0201] 0 placed hard macros will be treated like blockages.
```

Comment:
There are hard blockages that are not attached to a macro.  Are they considered?

Reply:
I forgot about them, I am adding them back.